src/patternfly/components/Pagination/pagination.scss

```diff
@@ -12,6 +13,7 @@
   --pf-c-pagination__nav--c-button--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-pagination__nav--c-button--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-pagination__nav--c-button--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-pagination--m-compact__nav--c-button--MarginLeft: var(--pf-global--FontSize--md);
```

Comment:
Why does this use a global FontSize var for the Margin?

Reply:
Doh! Because I copied and pasted the wrong thing :) Updated.